src/data-mgt/node/config/constants.js

```diff
@@ -1,109 +1,141 @@
 const axios = require("axios");
 const isEmpty = require("is-empty");
 const { logElement, logText, logObject } = require("../utils/log");
-const jsonify = require("../utils/jsonify");
 
 const devConfig = {
   MONGO_URI: "mongodb://localhost/",
   JWT_SECRET: process.env.JWT_SECRET,
   DB_NAME: process.env.MONGO_DEV,
   REDIS_SERVER: process.env.REDIS_SERVER_DEV,
```

Comment:
Hi @123MwanjeMike , we need to first update the ENVs in both the production and staging clusters before we merge this PR.

The updated .env was shared in the PR description....... cc: @NoahNsimbe